.github/workflows/ci.yml

```diff
@@ -7,6 +7,7 @@ on:
     branches:
       - main
       - development
+      - cbmc-proof
```

Comment:
It makes sense to add this, but please be careful to remove this when we merge this branch into main.

Reply:
Yes. I will.

Reply:
@yanjos-dev yes. I had suggested adding this so that we can make sure that in an attempt to fix errors found by CBMC we are not breaking the functionality/formatting etc. It would make merging the branches more simpler later on.